hooks/__init__.py

```diff
@@ -1,4 +1,4 @@
 """KPOps pre-commit hooks"""
 from pathlib import Path
 
-PATH_ROOT = Path(__file__).parents[1]
+PATH_ROOT: Path = Path(__file__).parents[1]
```

Comment:
why is the annotation necessary?

Reply:
Otherwise `pyright` wasn't recognizing that all the constants in `gem_docs.py` are of type `Path`